src/mpid/ch4/include/mpidpre.h

```diff
@@ -207,6 +207,10 @@ typedef struct MPIDIG_req_t {
     MPI_Datatype datatype;
 } MPIDIG_req_t;
 
+typedef struct MPIDIG_gpu_req_t {
+    void *stage_buf;
+} MPIDIG_gpu_req_t;
+
```

Comment:
@shawnccx this PR needs rebasing as new code has been merged to master. Since we need to do that I spotted this change that might have made sense when this PR was fatter than it is now. Probably this should be moved to your other OFI fallback PR. Would you mind removing this? Sorry for the back and forth.